mumd/src/network/udp.rs

```diff
@@ -148,10 +148,12 @@ async fn listen(
                             continue;
                         }
                     };
+                    state
+                        .lock()
+                        .unwrap()
+                        .broadcast_phase(StatePhase::UDPVoice);
```

Comment:
This isn't how the switching between UDP and TCP communication should be done. The protocol docs state that 
> The client starts the connectivity checks by sending a Ping packet to the server. When the server receives this packet it will respond by echoing it back to the address it received it from. Once the client receives the response from the server it can start using the UDP transport for audio data.

i.e. we should only start talking over UDP once we get a UDP ping response and not when we receive voice over UDP.